Sources/Overload/OvTools/include/OvTools/Utils/String.h

```diff
@@ -45,5 +52,25 @@ namespace OvTools::Utils
         * @param p_isAvailable (A callback that must returning true if the input string is available)
         */
         static std::string GenerateUnique(const std::string& p_source, std::function<bool(std::string)> p_isAvailable);
+
+		/**
+		* Trim whitespaces from a string in a desired direction (Leading, Trailing or both as TwoWays).
+        * @param p_str String to trim
+        * @param p_trimType The desired trim direction, default is Trailing
+		*/
+		static void Trim(std::string& p_str, const ETrimType p_trimType = ETrimType::Trailing);
+
+	private:
+		/**
+        * Leading trim of whitespaces from a string.
+        * param p_str String to trim
+		*/
+		static void LeadingTrim(std::string& p_str);
+
+		/**
+		* Trailing trim of whitespaces from a string.
+		* param p_str String to trim
+		*/
+		static void TrailingTrim(std::string& p_str);
```

Comment:
These don't need to be exposed in the header file. Keep them in the `.cpp` and remove `OvTools::Utils::String::` from the function name